src/battle-station.tsx

```diff
@@ -70,8 +70,91 @@ export const Dashboard: React.FC<DashboardProps> = ({screen, client, wss }) => {
     }
   }
 
+  const getBlockTransactions = useCallback( async (txHash: string, blockInfo: BlockDetails) => {
```

Comment:
is this callback misnamed? also got some phantom lines and missed spaces in here -- could use a standards pass